### PR TITLE
Support for running teuthology suites in OpenSUSE Leap images

### PR DESCRIPTION
**Pull request by rjfd**
This PR adds support in ceph-cm-ansible for deploying and run teuthology suites/jobs in OpenSUSE Leap images.

We tested this PR (thanks to @smithfarm) in sepia lab to ensure that no regression was being added
http://pulpito.ceph.com/smithfarm-2016-09-12_11:19:22-rados-jewel---basic-smithi/

### CODE REVIEW

**Comment by smithfarm** [comment]
Note: I also tested this in OVH.

**Comment by smithfarm** [comment]
I took a look at the two failed jobs in http://pulpito.ceph.com/smithfarm-2016-09-12_11:19:22-rados-jewel---basic-smithi/

The first one, job number 411592, looks like http://tracker.ceph.com/issues/16133

The second failure, job number 411594, is http://tracker.ceph.com/issues/16516

**Comment by smithfarm** [comment]
And here is the ceph/ceph-qa-suite branch (based on "jewel", with one commit on top) I used for the test run: https://github.com/ceph/ceph-qa-suite/tree/wip-ceph-cm-ansible-jewel

**Comment by rjfd** [comment]
@zmc can you check this PR? thanks!

File: roles/testnode/tasks/zypper_systems.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+# Tasks common to all systems that use the zypper
+# package manager
```
Comment (zmc) [inline roles/testnode/tasks/zypper_systems.yml]: Should we add a comment here along the lines of "this is mostly a copy of yum_systems.yml", just to make any decisions about later changes slightly easier?

**Comment by rjfd** [comment]
@zmc added the comment you suggested

**Comment by rjfd** [comment]
@zmc I think this is ready to merge!

**Comment by rjfd** [comment]
@zmc rebased on top of master. Could you merge this PR to get teuthology to work with opensuse, it's last piece that is missing.
Thanks.

**Comment by smithfarm** [comment]
@zmc @dmick Can we get a review?

File: roles/testnode/tasks/zypper_systems.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  tags:
+    - sudoers
+
+- name: Configure /etc/security/limits.conf
+  template:
+    src: limits.conf
+    dest: /etc/security/limits.conf
+    group: root
+    owner: root
+    mode: 0644
```
Comment (dmick) [inline roles/testnode/tasks/zypper_systems.yml]: If the only difference in the code was the mode, 0755 in the generic code looks like a bug

Comment (smithfarm) [inline roles/testnode/tasks/zypper_systems.yml]: @dmick The mode is set to 0644 in `roles/testnode/tasks/yum_systems.yml` as well.

Comment (dmick) [inline roles/testnode/tasks/zypper_systems.yml]: fair enough, just saying...the generic code probably would serve with the right mode

**Pull request merged**